cli/tests/test_abandon_command.rs

```diff
@@ -152,11 +152,6 @@ fn test_rebase_branch_with_merge() {
     ├─╯
     ◉  a b e??
     "###);
-
-    let stderr = test_env.jj_cmd_failure(&repo_path, &["abandon", "root()"]);
-    insta::assert_snapshot!(stderr, @r###"
```

Comment:
What happens now if we try to abandon the root commit?

Reply:
I moved all the tests of immutable commits into `test_immutable_commits.rs`. There is a test of `jj edit 'root()'` but I didn't test each command with the root commit; I only tested that they respect immutable commits in general.